File: graph/src/main/groovy/net/twisterrob/gradle/graph/vis/DummyTaskVisualizer.java
package net.twisterrob.gradle.graph.vis;

import java.util.Map;

import org.gradle.api.*;
import org.gradle.api.initialization.Settings;

import net.twisterrob.gradle.graph.tasks.*;

public class DummyTaskVisualizer implements TaskVisualizer {
	@Override public void showUI(Settings project) {
		// do nothing
	}
	@Override public void initModel(Map<Task, TaskData> graph) {
		// do nothing
	}
	@Override public void update(Task task, TaskResult result) {
		// do nothing
	}
	@Override public void closeUI() {
		// do nothing
	}
}

File: graph/src/main/kotlin/net/twisterrob/gradle/graph/vis/DummyTaskVisualizer.kt
package net.twisterrob.gradle.graph.vis

import net.twisterrob.gradle.graph.tasks.TaskData
import net.twisterrob.gradle.graph.tasks.TaskResult
import org.gradle.api.Task
import org.gradle.api.initialization.Settings

class DummyTaskVisualizer : TaskVisualizer {

	override fun showUI(settings: Settings) {
		// do nothing
	}

	override fun initModel(graph: Map<Task, TaskData>) {
		// do nothing
	}

	override fun update(task: Task, result: TaskResult) {
		// do nothing
	}

	override fun closeUI() {
		// do nothing
	}
}
